master/getting-started/mesos/index.md

```diff
@@ -29,12 +29,7 @@ and improve reliability. See additional information on
 
```

Comment:
Couple of comments about the image.

-  Do we need the comment about Marathon-LB using kernel routing?  And if so, why is it pointing at the link between the two?
-  Might be worth putting the second agent on there with the third Calico IP  (which is otherwise missing from the doc)